geniza/entities/models.py

```diff
@@ -276,3 +276,111 @@ class Meta:
 
     def __str__(self):
         return f"{self.type} relation: {self.from_person} and {self.to_person}"
+
+
+class Place(models.Model):
+    """A named geographical location, which may be associated with documents or people."""
+
+    names = GenericRelation(Name, related_query_name="place")
+    latitude = models.DecimalField(
+        max_digits=6,
+        decimal_places=4,
+        blank=True,
+        null=True,
+    )
+    longitude = models.DecimalField(
+        max_digits=7,
+        decimal_places=4,
+        blank=True,
+        null=True,
+    )
```

Comment:
I haven't used it before and am not sure how much it takes to integrate, but would it make sense to use a `PointField` from geodjango here? It seems like the default widgets might get you a lot of functionality that would be useful.

Reply:
I agree that it's probably not worth the overhead of the additional dependencies. It's possible that PUL already has postgis installed since they are doing some geospatial work (or perhaps they're just using Solr spatial search, which I haven't had the chance to work with but seems cool)  - I'd be happy to ask what they're using if that would be helpful.